README.md

```diff
@@ -1,20 +1,13 @@
-# ListView
+# Ember Collection
 
 [![Circle CI](https://circleci.com/gh/emberjs/ember-collection.svg?style=shield)](https://circleci.com/gh/emberjs/ember-collection)
 
-An efficient incremental rendering list view for large lists.
+An efficient incremental rendering component for large lists.
```

Comment:
might be nice to emphasize the layout aspects here too. i suspect many users will use this purely for that use case as a replacement for something like masonry.